pkg/fmtutil/fmtutil.go

```diff
@@ -45,13 +45,13 @@ func formatLabels(labels map[string]string) string {
 	return res
 }
 
-// formatSlice formats the slice with `--format` flag.
+// FormatSlice formats the slice with `--format` flag.
 //
 // --format="" (default): JSON
 // --format='{{json .}}': JSON lines
 //
-// formatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
-func formatSlice(cmd *cobra.Command, x []interface{}) error {
+// FormatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
+func FormatSlice(cmd *cobra.Command, x []interface{}) error {
```

Comment:
```suggestion
func FormatSlice(format string, x []interface{}) error {
```

to decouple `pkg/...` from CLI, or just retain this function under `cmd/nerdctl`

Reply:
Done(